CMSgov/onemac
# oy2-6318 Form Header Text

Story: https://qmacbis.atlassian.net/browse/OY2-6318

Deploy URL: https://dn6ofbibnmdyb.cloudfront.net/spa

Test Steps:

Login as stateuseractive@cms.hhs.local.
Goto SPA Form.
Verify Text based on story.

Changes
Added text based on story.
NOTE: The styling happens in Story: https://qmacbis.atlassian.net/browse/OY2-8952. Added color for title, but realized the arrow and the new route/flow may impact other styling items.
